flutter_maplibre_gl/analysis_options.yaml

```
@@ -0,0 +1 @@
+include: package:very_good_analysis/analysis_options.yaml
```

```suggestion
include: ../analysis_options.yaml
```

This should resolves the failing CI, too.